scripts/exglobal_archive.sh

```diff
@@ -155,10 +155,8 @@ if [[ ${HPSSARCH} = "YES" || ${LOCALARCH} = "YES" ]]; then
     if [[ "${mod}" -eq 0 ]] || [[ "${PDY}${cyc}" -eq "${firstday}" ]]; then SAVEFCSTIC="YES" ; fi
 
 
-    ARCH_LIST="${DATA}/archlist"
-    [[ -d ${ARCH_LIST} ]] && rm -rf "${ARCH_LIST}"
-    mkdir -p "${ARCH_LIST}"
-    cd "${ARCH_LIST}" || exit 2
+    [[ -d ${DATA} ]] && rm -rf "${DATA}"
```

Comment:
We don't want this line here (`[[ -d ${DATA} ]] && rm -rf "${DATA}"`) since it will remove the `${DATA}` folder created by `ush/jjob_header.sh` https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/jjob_header.sh#L66:L74).

At this point in the job we only need to `cd` into `${DATA}` (what happens on line 159 - `cd "${DATA}" || exit 2`) so let's remove line 158.

Reply:
Done.

Reply:
Thanks @HenryWinterbottom-NOAA !